src/style.css

```diff
@@ -1,3 +1,60 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
You should consider adding a fallback font-family in case the main one doesn't load. For example, 'Roboto, Arial, sans-serif'.